Core_Method/CoreMethod_FWD/ROS_FWD_Integrator.m

```diff
@@ -95,7 +95,8 @@
     global ros_A ros_C ros_M ros_E
     global ros_Alpha ros_Gamma
     global ros_NewF
-    
+    global istage    % Added by Arash to test JacVec convergence
+    istage=0;
```

Comment:
I believe these two lines can be safely deleted. We globalized this variable to take statistics for a paper.